pyccel/codegen/pipeline.py

```diff
@@ -159,7 +159,7 @@ def handle_error(stage):
         folder = os.path.abspath(folder)
 
     # Define directory name and path for pyccel & cpython build
-    pyccel_dirname = '__pyccel__'
+    pyccel_dirname = '__pyccel__'+os.environ.get('PYTEST_XDIST_WORKER', '')
```

Comment:
Could you please add empty spaces around the `+` binary operator?

Reply:
Fixed b245ece3